W-15577052 - RPA Manager release notes 2024-04/2. (#362)
mulesoft/docs-rpa: rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc

RPA Manager is the cloud-based control plane for your automation projects.

[[april-22-2024]]
== April 22, 2024

// == Fixed Issues
include::partial$release-notes/rn-headings.adoc[tag=fixedIssues,leveloffset=+2]

[%header,cols="6a,1a"]
|===
|Issue Resolution |ID

> | A long Japanese process name will now be displayed properly on a process tile. | W-14835485

Reviewer: ```suggestion
| Process names with Japanese characters no longer get split on a process tile. | W-14835485
```

> | Content of table cells no longer gets truncated when there's more space available. | W-15242583

Reviewer: ```suggestion
| The content of table cells no longer gets truncated when there's more space available. | W-15242583
```

> | The content of table cells will now be displayed properly with respect to the respective column width. | W-15242583

Reviewer: ```suggestion
| Content of table cells no longer gets truncated when there's more space available. | W-15242583
```

> | The text of downloadable applications on the RPA Manager's homepage will be shown correctly formatted. | W-15312305

Reviewer: ```suggestion
| The text of downloadable applications on the RPA Manager's homepage no longer extends outside the buttons. | W-15312305
```

> | The text of downloadable applications on the RPA Manager's homepage no longer extends outside the buttons. | W-15312305

Reviewer 2: ```suggestion
| The text of downloadable applications on the RPA Manager homepage no longer extends outside the buttons. | W-15312305
```

Reviewer 2: As per our [Style Guide](https://confluence.internal.salesforce.com/pages/viewpage.action?spaceKey=MTDT&title=MuleSoft+CX+Writing+Style+Reference), at "possessives (‘s)"

> | Now the field *To (email)* in an _Alert Target_ will be enforced as a mandatory field. | W-11716347

Reviewer: ```suggestion
| The field *To (email)* in an Alert Target is now enforced as a mandatory field. | W-11716347
```

> | The email address in an _Alert Target_ will be verified. | W-11720892

Reviewer: ```suggestion
| The email address in an Alert Target is now verified. | W-11720892
```

> | Modifying an inactive _Alert Rule_ will now keep the rule inactive. | W-15424278

Reviewer: ```suggestion
| Modifying an inactive Alert Rule no longer activates it. | W-15424278
```
